client/app/lib/components/table/TanStackTableBuilder/useTanStackTableBuilder.tsx

```diff
@@ -82,8 +86,25 @@ const useTanStackTableBuilder = <D extends object>(
   });
 
   const generateAndDownloadCsv = async (): Promise<void> => {
+    const downloadableColumns = (table.getHeaderGroups()[0]?.headers ?? [])
+      .map(({ column: { columnDef: { header } = {} } = {} }) => header)
+      .filter((header): header is string => isString(header))
+      .reduce<string[]>((acc, header, index) => {
+        if (getRealColumn(index)?.csvDownloadable ?? true) {
```

Comment:
Why change `.csvDownloadable` default value from `false` to `true`? This can lead to unintended side-effects?